recipes/squirrel-core/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "squirrel-core" %}
+{% set version = "0.12.0.dev8123371" %}
```

Comment:
This is a dev release and you would need a local config to send it to a dev channel. See https://github.com/conda-forge/cfep/blob/main/cfep-05.md for more info.

Is there a stable release you want to get first on the channel and work on the dev release later in the feedstock?

Reply:
Soon, the first non dev release will be published. Will upgrade the yaml before merging.

Reply:
Sounds good 😊

Reply:
Please ping us when you do so we can merge,